examples/games/breakout.rs

```diff
@@ -101,15 +98,16 @@ struct Brick;
 #[derive(Resource, Deref)]
 struct CollisionSound(Handle<AudioSource>);
 
-// This bundle is a collection of the components that define a "wall" in our game
-#[derive(Bundle)]
-struct WallBundle {
-    // You can nest bundles inside of other bundles like this
-    // Allowing you to compose their functionality
-    sprite: Sprite,
-    transform: Transform,
-    collider: Collider,
-}
+// Default is required to be part of the Required Components used with the Wall Component bellow
```

Comment:
- bellow -> below
- "Default is required" could be worder better to not overlap with required components
- I don't generally see a reason to capitalize common features like Component or Required Components
```suggestion
// Default must be implemented to define this as a required component for the Wall component below
```